cypress/support/commands/chainable.ts

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * Check if an element is visible to the user on the screen.
+ */
+Cypress.Commands.add('isVisible', { prevSubject: true }, (subject) => {
+  const bottom = Cypress.$(cy.state('window')).height();
+  const rect = subject[0].getBoundingClientRect();
+
+  expect(rect.top).not.to.be.greaterThan(bottom);
+  expect(rect.bottom).not.to.be.greaterThan(bottom);
+  expect(rect.top).not.to.be.greaterThan(bottom);
```

Comment:
aren't these duplicated or am I misreading this?

Reply:
Thanks, it's a typo, I've removed the duplicate rows.